internal/ld/ld.go

```diff
@@ -500,15 +528,23 @@ func (t tableData) Swap(i, j int) {
 
 const maxFlagKeysDisplayed = 50
 
-func (b BranchRep) PrintReferenceCountTable() {
-	data := tableData{}
+func (b BranchRep) CountByFlag(flags []string) map[string]int64 {
```

Comment:
Factored a bit of the `PrintReferenceCountTable` out into another method so it can be used for other purposes (i.e. checking if a flag has no references for extinctions)